src/config/scrivito.ts

```diff
@@ -4,23 +4,26 @@ import { getJrPlatformConfig } from '../privateJrPlatform/getJrPlatformConfig'
 import { getJrPlatformInstanceId } from '../privateJrPlatform/multiTenancy'
 
 export function configureScrivito(options?: { priority?: 'background' }) {
+  const tenant = scrivitoTenantId()
+  if (!tenant) return // Other layers (like vite.config.ts) should have thrown an error.
```

Comment:
* [ ] FYI The comment couples this implementation with other implementations ("layers") for no good reason. Remove? Throw?

Reply:
Throw is not an option, because it would stop the control flow and would never execute `getRootComponent()` from `main.tsx`.